OasisDEX/oasis-borrow
# made VaultActionInput USD compatible

# [Add variant to VaultActionInput to display USD values based on flag](https://app.shortcut.com/oazo-apps/story/4898/add-variant-to-vaultactioninput-to-display-usd-values-based-on-flag)

![image](https://user-images.githubusercontent.com/6871923/176021659-0643021e-5266-4c1e-90ea-2df9fd649927.png)


## Changes 👷‍♀️
  <Please add short list of changes>
- Made VaultActionInput accept currencyCode instead of token, so it can be either token or USD
  
## How to test 🧪
- Go to Auto Sell Form, see values correctly displayed in USD. Try with token on other views, no regression.

PS.: I'm not sure if I shall also add `formatFiatBalance` for value in the input or just add 'USD' as on design or just keep it way it is? - https://www.figma.com/file/kAPyiF2JpYre2pgxiYunEr/Oasis-Master?node-id=6576%3A43198
